coral-spark/src/main/java/com/linkedin/coral/spark/transformers/FallBackToLinkedInHiveUDFTransformer.java

```diff
@@ -66,15 +67,15 @@ protected SqlCall transform(SqlCall sqlCall) {
       throw new UnsupportedUDFException(operatorName);
     }
     final String viewDependentFunctionName = operator.getViewDependentFunctionName();
+    String versionedFunctionName = FunctionUtils.getVersionedFunctionName(viewDependentFunctionName, operatorName);
```

Comment:
Since we leverage `VersionedSqlUserDefinedFunction` jointly with this new code, is there a way to encapsulate all this functionality in this class?

Reply:
Moved to a new method `getVersionedViewDependentFunctionName` in `VersionedSqlUserDefinedFunction`.